processing/src/main/java/org/apache/druid/segment/column/SimpleColumnHolder.java

```diff
@@ -21,6 +21,7 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.base.Supplier;
+import org.apache.druid.java.util.common.logger.Logger;
```

Comment:
I think this is going to fail Travis for unused import

Reply:
oh no! I just pushed a commit to remove the import.